# yceffort (no-new-date)

ê¸°ê¸° ì‹œê°„ì— ì˜ì¡´ì ì¸ `new Date()`ë¥¼ ì‚¬ìš©í•˜ì§€ ë§ˆì„¸ìš”. ëŒ€ì‹  `ServerDate()` í•¨ìˆ˜ë¥¼ ë§Œë“¤ì–´ ì‚¬ìš©í•˜ì„¸ìš”.

## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js
// âŒ
new Date()
```

Examples of **correct** code for this rule:

```js
// ğŸ‘Œ
new Date('2022-01-01')
// ğŸ‘Œ
ServerDate()
```

## When Not To Use It

- `ServerDate()` í•¨ìˆ˜ê°€ ì—†ëŠ” ê²½ìš°
- ê¸°ê¸° ì‹œê°„ì— ì˜ì¡´í•´ë„ ìƒê´€ì—†ëŠ” ê²½ìš°

## Further Reading

https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Date
